src/vivarium_cluster_tools/psimulate/branches.py

```diff
@@ -100,6 +100,8 @@ def __iter__(self):
         for job_config in product(
             self._keyspace["input_draw"], self._keyspace["random_seed"], self.branches
         ):
+            if job_config[2] is None:
```

Comment:
If there are no branches, this now returns an empty dict instead of None, so we're dealing with consistent types.